core/domain/story_domain.py

```diff
@@ -861,7 +863,7 @@ def has_exploration(self, exp_id):
                 return True
         return False
 
-    def to_dict(self):
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
Remove the typing and add ignore to the callsite

Reply:
done!